tests/test_http.py

```diff
@@ -1053,7 +1053,10 @@ def test_add_fail_for_resumable(self):
     upload = MediaFileUpload(
         datafile('small.png'), chunksize=500, resumable=True)
     self.request1.resumable = upload
-    self.assertRaises(BatchError, batch.add, self.request1, request_id='1')
+    with self.assertRaises(BatchError) as batch_error:
+      batch.add(self.request1, request_id='1')
+    self.assertEqual(str(batch_error.exception), '<BatchError "Media requests cannot be used in a batch request.">')
```

Comment:
I'd like to test if `batch.add(self.request1, request_id='1')` raise `BatchError` here, it should not be `BatchError: <unprintable BatchError object>` instead of correct message.